test/conformance/testdata/ineligible_endpoints.yaml

```diff
@@ -105,10 +105,10 @@
   link: https://github.com/kubernetes/community/blame/master/contributors/devel/sig-architecture/conformance-tests.md#L64
 - endpoint: getFlowcontrolApiserverAPIGroup
   reason: depends on alpha feature
-  link: https://github.com/kubernetes/enhancements/blob/f16c4c7f1c9e28a3cc4bb4d0e6503efea2ae7987/keps/sig-api-machinery/20190228-priority-and-fairness.md
```

Comment:
We can still open this link.
Why do we need to update here?

Reply:
It is better to modify the files under the master branch

Reply:
I changed it back

Reply:
> We can still open this link.
> Why do we need to update here?

I changed it back, thanks for reminding